src/store/getters.ts

```diff
@@ -98,18 +98,18 @@ const getters: GetterTree<VQBState, any> = {
   /**
    * Return all available domains (including pipelines but excluding currentPipelineName)
    */
-  availablePipelines: (state: VQBState) =>
-    Object.keys(state.pipelines).filter((name: string) => name !== state.currentPipelineName),
-
+  availablePipelineNames: (state: VQBState) =>
+    Object.keys(state.pipelines)
+      .concat(state.domains)
+      .filter((name: string) => name !== state.currentPipelineName)
+      .sort((a, b) => a.localeCompare(b)),
```

Comment:
OK to do this concatenation here, directly in the store, but I think the name `availablePipelineNames` is not right.
This contains domains and pipeline names, not just pipeline names. It's a bit confusing in terms of types.

Since we call these both "datasets" in the UI (see https://github.com/ToucanToco/weaverbird/blob/28bdcc5e28210cb3dc5aeb02b7b72ec9352aecf1/src/components/stepforms/JoinStepForm.vue#L7) may we rename that getter `availableDatasetsNames`?